### ISSUE TITLE
When using `git spr`, the stack should be visible also when the discussion on the PR contains multiple messages, hiding older messages

### ISSUE THREAD

**Issue by schreter** [Reviewable/Reviewable#1060]
Currently, Reviewable hides older messages in the PR discussion, thus hiding the first message in the list, which contains the links to individual components of the `git spr` stack.

Either this first message with the stack should not be hidden at all (and, since it's the initial commit/PR message, it would probably be helpful in general to keep it visible) or at least the stack should be displayed, so it's easy to switch between stack items (currently, I have to open all discussions to get back to the first message, which costs multiple clicks).

**Comment by pkaminski** [comment]
Yup, we're likely moving towards giving PR information (including the description with the links) its own panel, separate from the auto-collapsed discussion.  The timeline is likely on the order of a 2-3 months if all goes as planned, but the design isn't finalized yet so it could get pushed out.
